erpnext/stock/doctype/item/item.js

```diff
@@ -216,6 +216,28 @@ frappe.ui.form.on("Item", {
 
 	set_meta_tags(frm) {
 		frappe.utils.set_meta_tag(frm.doc.route);
+	},
+
+	copy_from_item_page_template: function(frm) {
```

Comment:
Some thoughts on templating Item Pages:

This Page Builder template implementation is not specific to Item so it doesn't make sense to make it exclusive for Item.
I don't think it makes sense to make "templates" out of templates. Page Builder templates are templates by themselves. If you are making similar item pages, you can make one item page and just duplicate it.
Also, Page Builder templates should allow dynamic values, and at least provide the Item document as the context.
Now that I think about it, it makes sense to make a Print Format Builder alternative for any type of WebsiteGenerator document but for it's web view. This needs more thought.